setup.py

```diff
@@ -5,7 +5,7 @@
 except ImportError:
     from distutils.core import setup
 
-version = '0.4.2'
+version = '0.4.3'
```

Comment:
version 0.5.0 - this is a substantial change to the timestamps and is more than just a patch.